examples/Forms/TextInput.md

```diff
@@ -110,6 +109,48 @@ const [value, setValue] = useState('Some valid input');
 />
 ```
 
+Label example: The optional `errorMessage` prop is not set, and the 
+`hideError` prop is set to `true.` Remove the `errorMesage` style.
```

Comment:
Typo: `errorMessage`
I would also change this to "`hideError` prop is set to `true,` which removes the `errorMessage` style."

Reply:
What is the type?

Reply:
It says "errorMesage" instead of "errorMessage"

Reply:
oh i see